sympy/solvers/inequalities.py

```diff
@@ -126,7 +126,7 @@ def solve_poly_inequalities(polys):
     Union(Interval.open(-oo, -sqrt(3)), Interval.open(-1, 1), Interval.open(sqrt(3), oo))
     """
     from sympy import Union
-    return Union(*[solve_poly_inequality(*p) for p in polys])
+    return Union(*[Union(*solve_poly_inequality(*p)) for p in polys])
```

Comment:
Is this the same as `return Union(*[s for p in polys for s in solve_poly_inequality(*p)])`?

Reply:
Yes. Your version is better because Union processes all the args at once.